src/Microsoft.Dnx.Tooling/Publish/PublishProject.cs

```diff
@@ -107,6 +107,17 @@ private bool EmitNupkg(PublishRoot root)
                 root.Operations.Delete(TargetPath);
             }
 
+            // If this is a wrapper project, we need to generate a lock file before building it
+            if (IsWrappingAssembly())
+            {
+                var success = Restore(root, publishProject: this, restoreDirectory: project.ProjectDirectory)
```

Comment:
Do we want to suppress the output here? Can this project potentially reference things that aren't in the local set of packages copied?